src/coreComponents/physicsSolvers/solidMechanics/benchmarks/Sneddon-Validation.xml

```diff
@@ -187,14 +179,7 @@
     <PackCollection
       name="displacementJumpCollection"
       objectPath="ElementRegions/Fracture/embeddedSurfaceSubRegion"
-      fieldName="displacementJump"
-      minSetSize="120"/>
-      
-    <PackCollection
-      name="cellCentersCollection"
-      objectPath="ElementRegions/Fracture/embeddedSurfaceSubRegion"
-      fieldName="elementCenter"
-      minSetSize="120"/>      
+      fieldName="displacementJump" />
   </Tasks>
```

Comment:
no need for a min set size coz now the collection just gets resized every time the field changes size?

Reply:
Yes exactly. 

There can still be trailing 0s though since the HDF arrays aren't ragged / AoAs. Practically now we just expand the dataset as needed, which will add 0s to all previous 'rows', and we never contract the dataset: it will be sized to accommodate the largest set of collected data from the run.

Reply:
How hard would it be to write it out AoA style?

Reply:
IIRC it will require defining a couple data types, one for the values extracted by a single index, and another variable-length type using the first type. Not sure how it plays in a dataspace with an unlimited dimension. 

The bigger unknown is dealing with the write in parallel w/ MPIO since HDF5 is a bit cumbersome in that respect... it might handle variable-length writes from processes transparently, or might require explicitly being told everything about the write (which is mostly the case right now).

Reply:
I guess this will only happen if the collection has not been written to file yet. So, basically, if the field is resized in the same `packCollection` event before being written to file then all time-steps will have the field with the maximum size. But this should not happen if an output event occurs before the resizing. Am I wrong?

Reply:
A size increase at any point results in the dimensions of the hdf dataspace being changed, so even rows already written to file will have 0s appended when the extent is changed.

Correlating the rows and the mesh object coordinates should in most cases allow the determination of which elements are padding elements and which are not.